### PR TITLE
Organisators autosuggest fix

### PR DESCRIPTION
**Pull request by simong**

### CODE REVIEW

File: shared/gh/js/views/tenant-admin/gh.admin-edit-organiser.js [hunk 1]
```diff
@@ -210,32 +210,31 @@ define(['gh.core', 'jquery.jeditable'], function(gh) {
...
-                        });
-
-                        // If there is a relatedTarget set on the event that triggered the function, focus on it
-                        if ($(ev.relatedTarget).length) {
-                            $(ev.relatedTarget).focus();
-                        }
-                    }
-                }
+                // Submit the AutoSuggest field
+                submitAutoSuggest(this, original);
```
Comment (nicolaasmatthijs) [inline shared/gh/js/views/tenant-admin/gh.admin-edit-organiser.js]: `submitAutoSuggest` doesn't need `this` anymore

File: shared/gh/js/views/tenant-admin/gh.admin-edit-organiser.js [hunk 2]
```diff
@@ -210,32 +210,31 @@ define(['gh.core', 'jquery.jeditable'], function(gh) {
...
+                }
+            });
+
+            // Close the autosuggest when the location cell is focussed. Note that the `focusout`
+            // event on the input field cannot be used as there's no reliable way to determine where
+            // the user is moving their focus. For example, when clicking an autosuggest search result,
+            // the focusout event would trigger in Firefox. As Firefox doesn't pass the target DOM
+            // element in the event, there's no way of knowing whether an autosuggest search result
+            // is clicked or the user moved on to the location cell
+            $('.gh-event-location').on('focusin', closeAutoSuggest);
```
Comment (nicolaasmatthijs) [inline shared/gh/js/views/tenant-admin/gh.admin-edit-organiser.js]: The autosuggest is now not closing when you tab backwards from the event location. I wonder if it makes sense doing something along the following lines instead:

```
var closeAutosuggest = function(ev) {
  if (!$.contains($('.gh-event-organisers'), ev.target) {
     // ...
  }
};

$(document).on('click', closeAutosuggest);
$(document).on('focus / focusin', closeAutosuggest);
```

File: shared/gh/js/views/tenant-admin/gh.admin-edit-organiser.js [hunk 3]
```diff
@@ -210,32 +210,31 @@ define(['gh.core', 'jquery.jeditable'], function(gh) {
...
-                }
+                // Submit the AutoSuggest field
+                submitAutoSuggest(this, original);
+
+                // Track the user editing organisers
+                timeFromStart = (new Date() - timeFromStart) / 1000;
+                gh.utils.trackEvent(['Data', 'Lecturer edit', 'Completed'], {
+                    'time_from_start': timeFromStart
+                });
             };
```
Comment (nicolaasmatthijs) [inline shared/gh/js/views/tenant-admin/gh.admin-edit-organiser.js:222]: Can we removing threedotting from the list items in the admin UI? Isn't it much more usable when the full list of lecturers shows?

**Comment by nicolaasmatthijs** [comment]
Finished review. Back to @simong for follow-up

**Comment by simong** [comment]
Pushed some follow-up. I'm going to contact Margaret about the threedotting

**Pull request merged**